module/zfs/dsl_scan.c

```diff
@@ -234,7 +234,7 @@ static int zfs_resilver_disable_defer = B_FALSE;
 static int zfs_free_bpobj_enabled = 1;
 
 /* Error blocks to be scrubbed in one txg. */
-unsigned long zfs_scrub_error_blocks_per_txg = 1 << 12;
+uint_t zfs_scrub_error_blocks_per_txg = 1 << 12;
```

Comment:
```suggestion
static uint_t zfs_scrub_error_blocks_per_txg = 1 << 12;
```

This should really be static as well.

Reply:
done!